### PR TITLE
Added french documentation for upgrade

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | 
| License       | MIT

We forgot the french documentation for the upgrade doc. 

@Strubbl @jlnostr Need help for german translation :)

### CODE REVIEW

File: docs/fr/user/upgrade-2.0.x-2.1.1.rst [hunk 1]
```diff
@@ -28,25 +26,45 @@ Téléchargez la dernière version de wallabag :
 
 .. code-block:: bash
 
-    wget http://wllbg.org/latest-v2-package && tar xvf latest-v2-package
```
Comment (tcitworld) [inline docs/fr/user/upgrade-2.0.x-2.1.1.rst]: We lost wllbg.org redirection ?

File: docs/fr/user/upgrade-2.0.x-2.1.1.rst [hunk 2]
```diff
@@ -28,25 +26,45 @@ Téléchargez la dernière version de wallabag :
...
+    wget http://framabag.org/wallabag-release-2.1.1.tar.gz && tar xvf wallabag-release-2.1.1.tar.gz
```
Comment (tcitworld) [inline docs/fr/user/upgrade-2.0.x-2.1.1.rst]: Maybe a subdirectory here if you want many releases.

File: docs/fr/user/upgrade-2.0.x-2.1.1.rst (hunk 1 above)
Comment (nicosomb) [inline docs/fr/user/upgrade-2.0.x-2.1.1.rst]: This page is for the 2.0.x -> 2.1.1 upgrade. So we need to hardcode this version.

File: docs/fr/user/upgrade-2.0.x-2.1.1.rst (hunk 2 above)
Comment (nicosomb) [inline docs/fr/user/upgrade-2.0.x-2.1.1.rst]: One day... ;-)

File: docs/fr/user/upgrade-2.1.x-2.1.y.rst [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
-La dernière version de wallabag est publiée à cette adresse : https://www.wallabag.org/pages/download-wallabag.html. Pour mettre à jour votre installation de wallabag, exécutez les commandes suivantes dans votre répertoire d'installation (remplacez ``2.0.8`` par le numéro de la dernière version) :
+La dernière version de wallabag est publiée à cette adresse : https://www.wallabag.org/pages/download-wallabag.html. Pour mettre à jour votre installation de wallabag, exécutez les commandes suivantes dans votre répertoire d'installation (remplacez ``2.1.2`` par le numéro de la dernière version) :
 
 ::
 
     rm -rf var/cache/*
     git fetch origin
     git fetch --tags
-    git checkout 2.0.8
+    git checkout 2.1.2 --force
```
Comment (jlnostr) [inline docs/fr/user/upgrade-2.1.x-2.1.y.rst]: I think we're still on 2.1.1 :)

Comment (jlnostr) [inline docs/fr/user/upgrade-2.1.x-2.1.y.rst]: oops, accidentally hit the "approve" button 😀

Comment (nicosomb) [inline docs/fr/user/upgrade-2.1.x-2.1.y.rst]: It's to prepare the 2.1.2 (it's the 2.1.x -> 2.1.y file)

File: docs/de/user/upgrade-2.0.x-2.1.1.rst [hunk 4]
```diff
@@ -1,5 +1,5 @@
-Wallabag updaten
-================
+Wallabag von 2.0.x auf 2.1.1 updaten
+====================================
```
Comment (Strubbl) [inline docs/de/user/upgrade-2.0.x-2.1.1.rst]: looks false in here, but monospaced it looks ok.

File: docs/de/user/upgrade-2.0.x-2.1.1.rst [hunk 5]
```diff
@@ -30,23 +30,43 @@ Lade das neueste Release von wallabag herunter:
...
+        from_email: no-reply@wallabag.org
+        rss_limit: 50
+        rabbitmq_host: localhost
+        rabbitmq_port: 5672
+        rabbitmq_user: guest
+        rabbitmq_password: guest
+        redis_host: localhost
+        redis_port: 6379
+
+Du kannst `hier eine Dokumentation über die Parameter finden<http://doc.wallabag.org/en/master/user/parameters.html>`_.
```
Comment (Strubbl) [inline docs/de/user/upgrade-2.0.x-2.1.1.rst]: @nicosomb is a space needed here between the link name and its location in angle brackets?

**Pull request merged**